NetLicensingClient/RestController/NetLicensingAPI.cs

```diff
@@ -20,8 +20,11 @@ class NetLicensingAPI
     {
         public enum Method { GET, POST, DELETE };
 
-        public static netlicensing request(Context context, Method method, String path, Dictionary<String, String> parameters)
+        public static netlicensing request(Context context, Method method, String path, Dictionary<String, String> parameters, int timeoutInSeconds = 100)
         {
+            if (timeoutInSeconds <= 0 || timeoutInSeconds > 300)
```

Comment:
Where the 300 is coming from? Would it be too restrictive?

Reply:
Just a suggestion to avoid too high timeouts. I think 300 (5 minutes) should be more than enough.

Reply:
At the same time - this is still a limit, which will prevent setting higher values for other use cases and customers.
Having 100 as default value, the correct redefinition of this parameter is the responsibility of the developer.